chester/catalog-applications/chester-range.md

```diff
@@ -0,0 +1,241 @@
+app config interval-aggreg 300
+app config interval-report 1800
+```
+
+## Specific Commands
+
+:::info
+
+You can easily explore the whole command tree structure - start with the `help`
+command.
+
+:::
+
+:::caution
+
+To apply a new configuration, you need to call `config save`, which applies the
+new configuration parameters and reboots the device.
+
+:::
+
+Command to set **sample interval** in seconds:
+
+```
+app config interval-sample <1-86400>
+```
+
+Command to set **aggregate interval** in seconds:
+
+```
+app config interval-aggreg <1-86400>
+```
+
+Command to set **report interval** in seconds:
+
+```
+app config interval-report <30-86400>
+```
+
+```
+aggreg
+```
+
+## Firmware
+
+The latest firmware is available in Catalog Applications [Firmware
+chapter](index.md#application-firmware).
+
+## Example JSON Message
+
+In this example **JSON** you can see data from all three variants
+
+Every available sensor has one aggrevated measurement, which contains the
+minimum, maximum, average and median value.
+
+```json
+{
+  "message": {
+    "version": 1,
+    "sequence": 1,
+    "timestamp": 1685093572
+  },
+  "attribute": {
+    "vendor_name": "HARDWARIO",
+    "product_name": "CHESTER-M",
+    "hw_variant": "CDGLS",
+    "hw_revision": "R3.4",
+    "fw_name": "(unset)",
+    "fw_version": "(unset)",
+    "serial_number": "2159019054"
+  },
+  "system": {
+    "uptime": 49,
+    "voltage_rest": null,
+    "voltage_load": null,
+    "current_load": null
+  },
+  "network": {
+    "imei": 351358816140765,
+    "imsi": 901288910018953,
+    "parameter": {
+      "eest": 7,
+      "ecl": 0,
+      "rsrp": -78,
+      "rsrq": -5,
+      "snr": 8,
+      "plmn": 23003,
+      "cid": 1011233,
+      "band": 20,
+      "earfcn": 6447
+    }
+  },
+  "thermometer": {
+    "temperature": 24.93
+  },
+  "accelerometer": {
+    "acceleration_x": 0,
+    "acceleration_y": -0.23,
+    "acceleration_z": 9.65,
+    "orientation": 2
+  },
+  "hygrometer": {
+    "temperature": {
+      "events": [],
+      "measurements": [
+        {
+          "timestamp": 1685093569,
+          "min": 24.9,
+          "max": 25.03,
+          "avg": 24.99,
+          "mdn": 25.01
+        }
+      ]
+    },
+    "humidity": {
+      "measurements": [
+        {
+          "timestamp": 1685093569,
+          "min": 35.18,
+          "max": 35.81,
+          "avg": 35.45,
+          "mdn": 35.36
+        }
+      ]
+    }
+  },
+  "w1_thermometers": [
+    {
+      "serial_number": 222768959,
+      "measurements": [
+        {
+          "timestamp": 1685093569,
+          "min": 24.31,
+          "max": 24.31,
+          "avg": 24.31,
+          "mdn": 24.31
+        }
+      ]
+    },
+    {
+      "serial_number": 222690915,
+      "measurements": [
+        {
+          "timestamp": 1685093569,
+          "min": 27,
+          "max": 27.43,
+          "avg": 27.22,
+          "mdn": 27.25
+        }
+      ]
+    }
+  ],
+  "sonar": {
```

Comment:
I would swap the `sonar` part to be before the `w1_thermometers` part as it is more significant to the application. I would prefer naming it `ultrasonic_ranger` instead of `sonar`.